django_meili/models.py

```diff
@@ -165,6 +170,11 @@ def meili_serialize(self):
             )
         )[0]
 
+        if getattr(self.MeiliMeta, "include_pk_in_search", False):
```

Comment:
We can just do something like
```python
serialized_model["fields"][self.MeiliMeta.primary_key] = getattr(self, self.MeiliMeta.primary_key)
```

Reply:
I didnt check if primary_key had a default, but sure we can do that

Reply:
✅